src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
+@import '../styles/utils/variables';
+@import '../styles/blocks/card';
+@import '../styles/blocks/star';
+
+body, h2, p {
```

Comment:
Consider moving resets and general styles to a separate file to keep your main file clean and clear